### PR TITLE
Update to Cadence v1.0.0-preview.28

### PR DESCRIPTION
**Pull request by ianthpun**
## Description

Automatically update to:
- [onflow/cadence v1.0.0-preview.28](https://github.com/onflow/cadence/releases/tag/v1.0.0-preview.28)
- [onflow/flow-go-sdk v1.0.0-preview.28](https://github.com/onflow/flow-go-sdk/releases/tag/v1.0.0-preview.28)
- [onflow/flow-go 26cb8ee404367dd294198c60092305b3f5fd22e5](https://github.com/onflow/flow-go/commit/26cb8ee404367dd294198c60092305b3f5fd22e5)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -92,11 +94,9 @@ require (
 	github.com/hashicorp/hcl v1.0.0 // indirect
 	github.com/holiman/bloomfilter/v2 v2.0.3 // indirect
 	github.com/holiman/uint256 v1.2.4 // indirect
-	github.com/huandu/go-clone v1.6.0 // indirect
-	github.com/huandu/go-clone/generic v1.7.2 // indirect
 	github.com/inconshreveable/mousetrap v1.1.0 // indirect
 	github.com/ipfs/bbloom v0.0.4 // indirect
-	github.com/ipfs/boxo v0.17.1-0.20240131173518-89bceff34bf1 // indirect
+	github.com/ipfs/boxo v0.19.0 // indirect
```
Comment (turbolent) [inline go.mod:99]: This shouldn't be updated, the old version is the correct one. See https://github.com/onflow/flow-go/pull/5822. The update to 0.19 is causing the need to update to Go 1.21

**Review by turbolent — COMMENTED** [review]
Thanks for helping with updating all the dependencies!

Just one issue with the unrelated dependency / Go update

File: go.mod [hunk 2]
```diff
@@ -12,11 +14,11 @@ require (
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 	github.com/improbable-eng/grpc-web v0.15.0
 	github.com/logrusorgru/aurora v2.0.3+incompatible
-	github.com/onflow/cadence v1.0.0-preview.26
+	github.com/onflow/cadence v1.0.0-preview.28
 	github.com/onflow/crypto v0.25.1
 	github.com/onflow/flow-core-contracts/lib/go/templates v1.0.0
-	github.com/onflow/flow-go v0.34.0-crescendo-preview.18
-	github.com/onflow/flow-go-sdk v1.0.0-preview.26
+	github.com/onflow/flow-go v0.34.0-crescendo-preview.16
```
Comment (turbolent) [inline go.mod:20]: This is downgrading flow-go, but shouldn't

**Comment by ianthpun** [comment]
@turbolent i wonder why its downgrading some things, is it because im giving it the wrong `flow-go` commit reference? [26cb8ee404367dd294198c60092305b3f5fd22e5](https://github.com/onflow/flow-go/commit/26cb8ee404367dd294198c60092305b3f5fd22e5)

**Comment by ianthpun** [comment]
gonna try it again with 4656ff2384409535461abd3e24ad9a2ce128dab7

**Comment by turbolent** [comment]
Yeah, need at least [4656ff2](https://github.com/onflow/flow-go/commit/4656ff2384409535461abd3e24ad9a2ce128dab7), i.e. what updated flow-go to that Cadence release (in https://github.com/onflow/flow-go/pull/5930)

**Pull request closed without merging**